src/container/srv_target.c

```diff
@@ -123,7 +123,10 @@ ds_get_cont_props(struct cont_props *cont_props, struct ds_iv_ns *pool_ns,
 	daos_prop_t	*props;
 	int		 rc;
 
-	props = daos_prop_alloc(7);
+	/* The provided prop entry types should cover the types used in
+	 * daos_props_2cont_props().
+	 */
+	props = daos_prop_alloc(9);
```

Comment:
can we use DAOS_PROP_CO_MAX here?

Reply:
DAOS_PROP_CO_MAX will cause getting quite some other extra info that is not needed for cont_props now and export some internal info.